tests/doxygen/bootloader.md

```diff
@@ -0,0 +1,24 @@
+Bootloader
+================
+
+[TOC]
+
+# Overview {#bootloader-overview}
+
+The bootloader is used to flash new firmware onto the Device, without having
+to use extra hardware like a
+[PICkit3](http://www.microchip.com/Developmenttools/ProductDetails.aspx?PartNO=PG164130)
+. This makes it easier for end users to upgrade their devices.
+
+The bootloader uses the <a
+[Device Firmware Upgrade (DFU)]
+(http://www.usb.org/developers/docs/devclass_docs/DFU_1.1.pdf)
+protocol. This is a standard protocol for upgrading firmware over USB.
+
+@section bootloader_procedure Update Procedure
+
+Generally the update procedure follows:
```

Comment:
This feels like it's missing a word or something. procedure involves the following steps perhaps?